index.html

```diff
@@ -7,17 +7,33 @@
   <link rel="stylesheet" href="css/index.css">
 </head>
 
+
 <body>
 
+  <header>
+
+    <ul>
+      <li><a href="#about">About</a></li>
+      <li><a href="#experience">Experience</a></li>
+      <li><a href="#skills">Skills</a></li>
+      <li><a href="#connect">Connect</a></li>
+      <li><a href="#message">Messages</a></li>
+
+    </ul>
```

Comment:
This change is not necessary for the rubric, but I noticed that on smaller screens/window sizes, the navbar becomes squished. You may want to try to play around with margin/padding/font-sizes or how you layout the navbar in your media queries for the smaller screen sizes to make it a bit neater:



<img width="884" alt="Screen Shot 2023-01-04 at 9 47 01 AM" src="https://user-images.githubusercontent.com/9094098/210585894-42790d47-a620-447f-b749-c1a0a619802e.png">